
---- EXPECTED OUTPUT
#0  c() called at [%sdebug_print_backtrace_limit.php:7]
#0  c() called at [%sdebug_print_backtrace_limit.php:7]
#1  b() called at [%sdebug_print_backtrace_limit.php:3]
#0  c() called at [%sdebug_print_backtrace_limit.php:7]
#1  b() called at [%sdebug_print_backtrace_limit.php:3]
#2  a() called at [%sdebug_print_backtrace_limit.php:17]
#0  c() called at [%sdebug_print_backtrace_limit.php:7]
#1  b() called at [%sdebug_print_backtrace_limit.php:3]
#2  a() called at [%sdebug_print_backtrace_limit.php:17]
---- ACTUAL OUTPUT
Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory
#0  c() called at [/root/php-src/Zend/tests/debug_print_backtrace_limit.php:7]
#0  c() called at [/root/php-src/Zend/tests/debug_print_backtrace_limit.php:7]
#1  b() called at [/root/php-src/Zend/tests/debug_print_backtrace_limit.php:3]
#0  c() called at [/root/php-src/Zend/tests/debug_print_backtrace_limit.php:7]
#1  b() called at [/root/php-src/Zend/tests/debug_print_backtrace_limit.php:3]
#2  a() called at [/root/php-src/Zend/tests/debug_print_backtrace_limit.php:17]
#0  c() called at [/root/php-src/Zend/tests/debug_print_backtrace_limit.php:7]
#1  b() called at [/root/php-src/Zend/tests/debug_print_backtrace_limit.php:3]
#2  a() called at [/root/php-src/Zend/tests/debug_print_backtrace_limit.php:17]
---- FAILED
